src/core/Object3D.js

```diff
@@ -702,6 +702,22 @@ Object3D.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 
 		}
 
+		if ( this.isScene ) {
+
+			if ( this.background && this.background.isColor ) {
+
+				object.background = this.background.getHex();
+
+			} else if ( this.background && this.background.isTexture ) {
+
+				object.background = serialize( meta.textures, this.background );
+
+			}
+
+			if ( this.environment && this.environment.isTexture ) object.environment = serialize( meta.textures, this.environment );
```

Comment:
The check for `.isTexture` is not necessary since `Scene.environment` only accepts textures right now.